src/config.ts

```
@@ -398,6 +469,60 @@ export async function loadFabricloggerConfig(flags: CliFlags, dryRun: boolean =
         return val;
     };
 
+    const parsePrometheusEndpointsConfig = (
+        value: (DeepPartial<PrometheusEndpointConfigSchema> | undefined)[] | undefined
+    ): ScraperEndpoint[] => {
+        const prometheusEndpoints = [];
+        if (value) {
+            for (const prometheusEndpointConfig of value) {
+                if (prometheusEndpointConfig && prometheusEndpointConfig?.url) {
+                    const parsedPrometheusEndpoint = {
+                        scrapeInterval: parseDuration(prometheusEndpointConfig.scrapeInterval),
+                        scrapeOptions: {
+                            url: prometheusEndpointConfig.url,
+                            timeout: prometheusEndpointConfig.timeout,
+                            allowCompression: prometheusEndpointConfig.allowCompression,
+                            validateContentType: prometheusEndpointConfig.validateContentType,
+                            validateCertificate: prometheusEndpointConfig.validateCertificate,
+                            userAgent: prometheusEndpointConfig.userAgent,
+                        },
+                    };
+                    prometheusEndpoints.push(parsedPrometheusEndpoint);
+                }
+            }
```

We should emit a warning stating that we're ignoring a configured endpoint and why. Alternatively we could also throw an error.